sopel/loader.py

```diff
@@ -51,6 +51,10 @@ def clean_callable(func, config):
         func.rate = getattr(func, 'rate', 0)
```

Comment:
Is it time to remove the `rate` attribute from callables and replace it with `func.user_rate`, for consistency?

Do we even have to consider the actual attributes of a plugin callable as part of the API? I hope not, because this is not something I want to go through a deprecation cycle for.

Reply:
And it is done.